specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2018-06-01-preview/apimcaches.json

```diff
@@ -0,0 +1,450 @@
+            "description": "The cache details were successfully updated."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./apimanagement.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "Caches"
+        ],
+        "operationId": "Cache_Delete",
+        "description": "Deletes specific Cache.",
+        "x-ms-examples": {
+          "ApiManagementDeleteCache": {
+            "$ref": "./examples/ApiManagementDeleteCache.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "./apimanagement.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ServiceNameParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/CacheIdParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/IfMatchRequiredParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The Cache was successfully deleted."
+          },
+          "204": {
+            "description": "The Cache was successfully deleted."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./apimanagement.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "CacheCollection": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CacheContract"
+          },
+          "description": "Page values."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Next page link if any."
+        }
+      },
+      "description": "Paged Caches list representation."
+    },
+    "CacheContract": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/CacheContractProperties",
+          "description": "Cache properties details."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./apimanagement.json#/definitions/Resource"
+        }
+      ],
+      "description": "Cache details."
+    },
+    "CacheContractProperties": {
+      "properties": {
+        "description": {
+          "type": "string",
+          "description": "Cache description",
+          "maxLength": 2000
+        },
+        "connectionString": {
+          "type": "string",
+          "description": "Runtime connection string to cache",
```

Comment:
This connectionString doesn't contain secrets, does it?

Reply:
yes it could.

---
In reply to: [261059399](https://github.com/Azure/azure-rest-api-specs/pull/5260#discussion_r261059399) [](ancestors = 261059399)

Reply:
Thanks for confirming that the secrets are not returned on GET